src/Sql/Sql.Management.Sdk/README.md

```diff
@@ -26,7 +26,7 @@ use-extension:
 
 ###
 ``` yaml
-commit: b7b77b11ba1f6defc86d309d4ca0d51b2a2646a7
+commit: a21edd5a33e0db83bde8d1dfe706a0cf3db6409e
```

Comment:
Is this commit Id okay?

Reply:
You can verify the generated SDK files and see if they are what you want. The SDK files will be different based on different swagger.

Reply:
Yes for my change they are good, I was wandering if I can regress some other feature.